src/recipe/recipe.controller.spec.ts

```diff
@@ -1,18 +1,169 @@
 import { Test, TestingModule } from '@nestjs/testing';
+import { RecipeService } from './recipe.service';
 import { RecipeController } from './recipe.controller';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { faker } from '@faker-js/faker';
 
-describe('RecipeController', () => {
-  let controller: RecipeController;
+describe("Recipe Controller's tests", () => {
```

Comment:
I'd leave `RecipeController`:

when you run the tests you will have something like this:

```
RecipeController
  should do this
  should do that
```

we're testing RecipeController here, not RecipeController's tests 😄